doozer/doozerlib/distgit.py

```diff
@@ -1094,6 +1096,7 @@ def wait(n):
                         record["nvrs"] = build_info["nvr"]
                     if not dry_run:
                         self.update_build_db(True, task_id=task_id, scratch=scratch)
+                        self.update_konflux_db(build_info, KonfluxBuildOutcome.SUCCESS, job_url=task_url, scratch=scratch)
```

Comment:
We should have two urls:
1. The ART job (Jenkins for the time being): `art_job_url`
2. The pipeline URL (brew today, konflux tomorrow): `build_pipeline_url`

Reply:
Done in 56aeef16c5c9442e469e0ca687f2a07b018f1539